src/styles/_catalog.scss

```diff
@@ -1,16 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $heightCard);
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
```

Comment:
1. no need to specify max-width;
2. you should use justify-content: center here, not margins (also you can't use margins for BEM block)

Reply:
fixed